AutowareAuto/src/common/autoware_auto_common/include/common/types.hpp

```diff
@@ -120,6 +120,29 @@ struct COMMON_PUBLIC PointXYZI
   }
 };
 
+#pragma pack(push,1)
+struct COMMON_PUBLIC PointXYZIRing
```

Comment:
Additionally, can we include some text in the PR description regarding why this new `PointXYZIRing` struct had to be defined, and why it had to replace the usage of `PointXYZIF` in the point_cloud_fusion package?

Reply:
Done!